src/screens/Assistant/Results.tsx

```diff
@@ -79,7 +81,9 @@ const Results: React.FC<Props> = ({
     return (
       <View style={styles.container}>
         <MessageBox>
-          <ThemeText style={styles.infoBoxText}>
+          <ThemeText
+            style={{...styles.infoBoxText, color: theme.status.info.main.color}}
```

Comment:
I think maybe text color actually gets cascaded? Could wrap ThemeText on top? cc @cbrevik

Reply:
I'll look into this

Reply:
It is not cascading from one `ThemeText` to the next.

This code which is the default style in `ThemeText` sets the text color if it is not overridden by specifying color in the `style` parameter.
```
const typeStyle = {
  ...theme.text[fontType],
  color: theme.text.colors[color],
};
```

Reply:
I experimented a little, but could not find a satisfying solution. It may be possible if investing a little more time, but I figured it was not worth it now. This means that if using `children` instead of `message` the `ThemeText` needs to set the color style.